test/libyul/yulOptimizerTests/ssaPlusCleanup/control_structures.yul

```diff
@@ -17,19 +17,19 @@
 //         let length_1 := mload(from)
 //         length := length_1
 //         mstore(to, length_1)
-//         let from_1 := add(from, 0x20)
-//         let to_1 := add(to, 0x20)
+//         let from_2 := add(from, 0x20)
+//         let to_3 := add(to, 0x20)
```

Comment:
Why do these indices start with 2 and 3?

Reply:
Yeah, I will split that out into a different PR. This is #5074

Reply:
I think it's fine in the same PR.